src/containerapp/HISTORY.rst

```diff
@@ -2,13 +2,17 @@
 
 Release History
 ===============
-upcoming
+incoming
 ++++++
 * 'az containerapp create': Support --registry-identity to use Environment identity for ACR authentication.
 * 'az containerapp job create': Support --registry-identity to use Environment identity for ACR authentication.
 * 'az containerapp registry set': Support --identity to use Environment identity for ACR authentication.
 * 'az containerapp job registry set': Support --identity to use Environment identity for ACR authentication.
 
+0.3.55
+++++++
+* 'az containerapp sessionpool create/update': Add NodeLTS container-type.
+
```

Comment:
Please pull the latest code. The version `0.3.55` has been released.

Reply:
You need to update to use `0.3.56`. Please append your change description behind others.